lib/breakers/dashboard.rb

```diff
@@ -0,0 +1,53 @@
+require 'sinatra/base'
+require 'json'
+
+require 'byebug'
+
+module Breakers
+  class Dashboard < Sinatra::Base
```

Comment:
I think that rather than using Sinatra here we should look to see if we can do this as a Rails engine maybe? And only include rails then as a developer dependency if one wants to build the engine.